pkg/archive/archive.go

```diff
@@ -115,6 +115,10 @@ const (
 	modeISSOCK = 0140000 // Socket
 )
 
+const (
+	paxSchilyXattr = "SCHILY.xattrs."
```

Comment:
@kstruczy according to golang src code - [archive/tar/common.go#L110](https://github.com/golang/go/blob/bad6b6fa9190e9079a6d6958859856a66f0fab87/src/archive/tar/common.go#L110), I think we should use "SCHILY.xattr." but not "SCHILY.xattrs." here?

Reply:
That's a good spot, I will fix it. Thanks